client/auth/login.jsx

```diff
@@ -1,214 +0,0 @@
-import SelfHostedInstructions from './self-hosted-instructions';
-import WordPressLogo from 'calypso/components/wordpress-logo';
-import { recordGoogleEvent } from 'calypso/state/analytics/actions';
-import { localizeUrl } from 'calypso/lib/i18n-utils';
-import * as OAuthToken from 'calypso/lib/oauth-token';
-import { errorTypes, makeAuthRequest, bumpStats } from './login-request';
-
-const debug = debugFactory( 'calypso:oauth' );
-
-export class Auth extends Component {
-	state = {
-		login: '',
-		password: '',
-		auth_code: '',
-		requires2fa: false,
-		inProgress: false,
-		errorLevel: false,
-		errorMessage: false,
-	};
-
-	getClickHandler = ( action ) => () =>
-		this.props.recordGoogleEvent( 'Me', 'Clicked on ' + action );
-
-	getFocusHandler = ( action ) => () =>
-		this.props.recordGoogleEvent( 'Me', 'Focused on ' + action );
-
-	focusInput = ( input ) => {
-		if ( this.state.requires2fa && this.state.inProgress === false ) {
-			input.focus();
-		}
-	};
-
-	handleAuthError = ( error ) => {
-		const stateChanges = {
-			errorLevel: 'is-error',
-			requires2fa: false,
-			inProgress: false,
-			errorMessage: error.message,
-		};
-
-		debug( 'Error processing login: ' + stateChanges.errorMessage );
-
-		bumpStats( error );
-
-		if ( error.type === errorTypes.ERROR_REQUIRES_2FA ) {
-			stateChanges.requires2fa = true;
-			stateChanges.errorLevel = 'is-info';
-		} else if ( error.type === errorTypes.ERROR_INVALID_OTP ) {
-			stateChanges.requires2fa = true;
-		}
-
-		this.setState( stateChanges );
-	};
-
-	handleLogin = ( response ) => {
-		debug( 'Access token received' );
-
-		bumpStats();
-
-		OAuthToken.setToken( response.body.access_token );
-
-		document.location.replace( '/' );
-	};
-
-	submitForm = async ( event ) => {
-		event.preventDefault();
-		event.stopPropagation();
-
-		this.setState( { inProgress: true, errorLevel: false, errorMessage: false } );
-
-		try {
-			const { login, password, auth_code } = this.state;
-			const response = await makeAuthRequest( login, password, auth_code.replace( /\s/g, '' ) );
-
-			this.handleLogin( response );
-		} catch ( error ) {
-			this.handleAuthError( error );
-		}
-	};
-
-	toggleSelfHostedInstructions = () => {
-		const isShowing = ! this.state.showInstructions;
-		this.setState( { showInstructions: isShowing } );
-	};
-
-	handleChange = ( event ) => {
-		const { name, value } = event.currentTarget;
-
-		if ( name === 'auth_code' ) {
-			this.setState( { auth_code: value } );
-			return;
-		}
-
-		this.setState( { [ name ]: value } );
-	};
-
-	canSubmitForm() {
-		// No submission until the ajax has finished
-		return ! this.state.inProgress;
-	}
-
-	render() {
-		const { translate } = this.props;
-		const { requires2fa, inProgress, errorMessage, errorLevel, showInstructions } = this.state;
-
-		return (
-			<Main className="auth">
-				<div className="auth__content">
-					<WordPressLogo />
-					<form className="auth__form" onSubmit={ this.submitForm }>
```

Comment:
Just realized there are some related `z-index` definitions we could cleanup from [this file](https://github.com/Automattic/wp-calypso/blob/trunk/client/assets/stylesheets/shared/functions/_z-index.scss#L70).

Reply:
Fixed in e062796.